cmd/version.go

```diff
@@ -26,7 +26,7 @@ import (
 	"github.com/spf13/cobra"
 )
 
-const version string = "v0.1.2"
+var version string = "DEV-BUILD"
```

Comment:
Shouldn't the version be in `main`?  The CLI's version isn't that interesting, but the version of whatever it does _is_.